### PR TITLE
Wip solution

### PR DESCRIPTION
**Pull request by drakeltheryuujin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,38 @@
...
 
--SWBAT 1
--SWBAT 2
+1. Practice writing JavaScript classes that inherit from other classes
 
 ## Introduction
 
-## SWBAT 1
+In this lab we're going to create a geometry application that allows us
+calculate different properties of multiple shape. We'll be using JavaScript's
```
Comment (maxwellbenton) [inline README.md]: us to calculate

File: README.md [hunk 2]
```diff
@@ -1,16 +1,38 @@
...
+`class` keyword to build objects that inherit from its base object.
 
-## SWBAT 2
+## Practice Writing JavaScript classes That Inherit From Other Classes
+
+1. Define a `Polygon` class. `Polygon` will accept an `Array` with parameter
```
Comment (maxwellbenton) [inline README.md]: something is out of place here "an 'Array' with parameter with integers'

File: README.md [hunk 3]
```diff
@@ -1,16 +1,38 @@
...
+with integers, which will represent each _side_ of a shape. Implement a `get`
+method called `getCount()` that counts the number of sides (each index in the array).
```
Comment (maxwellbenton) [inline README.md]: Maybe split these steps into bullets under 1.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,38 @@
...
+2. Define a `Triangle` class that inherits from `Polygon`. It has access to calculate
+`perimeter()` inherited from `Polygon`. Implement an `isValid()` method that checks if
+the the given 3 sides for a triangle is valid.
+
+> The sum of the lengths of any two sides of a triangle is greater than the length of
+the third side. If you take the three sides of a triangle and add them in pairs, the sum
+is greater than (not equal to) the third side. If that is not true, then it is not possible
+to construct a triangle with the given side lengths.
+
+3. Define a `Square` object that inherits from `Polygon`. It has access to calculate
```
Comment (maxwellbenton) [inline README.md]: It has access to `perimeter()`

File: README.md [hunk 5]
```diff
@@ -1,16 +1,38 @@
...
+`perimeter()` inherited from `Polygon`. Implement an `isValid()` method that checks if
+the the given 4 sides for a square is valid. A square is valid when the lengths of all sides
+are equal. Implement the method `area()` that calculates the area of the square.
```
Comment (maxwellbenton) [inline README.md]: Good. I think though that there still should be at least two child classes to illustrate how children might have custom methods but share parent methods

Comment (drakeltheryuujin) [reply README.md]: Will add maybe `rectangle` or `quadrilateral` to the next iteration.

**Pull request merged**